tools/buildscripts/build.py

```diff
@@ -160,36 +160,48 @@ def package_release(is_dev_release: bool):
                 if file.lower() == "dwmapi.dll":
                     dwmapi_dll_path = os.path.join(root, file)
 
-        # main dll
-        shutil.copy(ue4ss_dll_path, staging_dir)
+        # Create the ue4ss folder in staging_dir
+        ue4ss_dir = os.path.join(staging_dir, 'ue4ss')
+        os.makedirs(ue4ss_dir, exist_ok=True)
 
-        # proxy
+        # Move all files from assets folder to the ue4ss folder except dwmapi.dll
+        for root, _, files in os.walk('assets'):
+            for file in files:
+                if file.lower() != 'dwmapi.dll':
+                    src_path = os.path.join(root, file)
+                    dst_path = os.path.join(ue4ss_dir, os.path.relpath(src_path, 'assets'))
+                    os.makedirs(os.path.dirname(dst_path), exist_ok=True)
+                    shutil.copy(src_path, dst_path)
+
+        # Main dll and pdb
+        shutil.copy(ue4ss_dll_path, ue4ss_dir)
+
+        # Proxy
         shutil.copy(dwmapi_dll_path, staging_dir)
 
         if is_dev_release:
-            shutil.copy(ue4ss_pdb_path, staging_dir)
-            if os.path.exists(os.path.join(staging_dir, 'docs')):
-                shutil.copytree('docs', os.path.join(staging_dir, 'docs'))
+            shutil.copy(ue4ss_pdb_path, ue4ss_dir)
+            if os.path.exists(os.path.join(scan_start_dir, 'docs')):
+                shutil.copytree('docs', os.path.join(ue4ss_dir, 'Docs'))
```

Comment:
Is this capitalization an erroneous change ?

Reply:
No, all the other folders are capitalized and it just looks out of place not being so, and I don't see any harm in it.

Reply:
Two things:
1. Will this work on Linux ? Because names are case-sensitive on Linux.
2. The name was only changed in the call to copy the directory, not the call above to see if the directory exists, this alongside the fact that the real directory is all lowercase is what made this stand out to me as a potential error.

Reply:
If checks if the docs directory exists (it probably doesn't need to since we know it does, but better safe than sorry), then copies the docs directory to the new location and changes the name to have a capital D. I have no reason to suspect this wouldn't work on Linux, though the build scripts may need to be updated to work on Linux when the Linux port gets to that point.

Reply:
Okay I understand what it does now.
I misunderstood earlier.
This potential problem is a non-problem, no further discussion is needed.
I did test `os.path.exists` just out of curiosity, and it fails if the casing is wrong on Linux, just for future reference.